README.md

```diff
@@ -389,6 +389,10 @@ Board optional configurations.
 * `simulatorFirmwareVersion` {String} - Allows the simulator to use firmware version 2 features. (2 Possible Options)
   * `v1` - Firmware Version 1 (Default)
   * `v2` - Firmware Version 2
+* `simulatorFragmentation` {String} - Specifies how to break packets to simulate fragmentation. (3 Possible Options)
```

Comment:
I think another approach to have is be able to set the latency timer and/or serial input buffer size. On my computer i generally run with a 64 byte buffer to get time syncing working well.